### ISSUE TITLE
Angular 10 : Cannot find name Buffer

### ISSUE THREAD

**Issue by aviralgoyal** [gitbrent/PptxGenJS#812]
Just migrated to Angular 10 and I'm getting this error on compilation.

```
ERROR in node_modules/pptxgenjs/types/index.d.ts:69:50 - error TS2591: Cannot find name 'Buffer'. Do you need to install type definitions for node? Try `npm i @types/node` and then add `node` to the types field in your tsconfig.

69  stream(): Promise<string | ArrayBuffer | Blob | Buffer | Uint8Array>
                                                    ~~~~~~
node_modules/pptxgenjs/types/index.d.ts:75:88 - error TS2591: Cannot find name 'Buffer'. Do you need to install type definitions for node? Try `npm i @types/node` and then add `node` to the types field in your tsconfig.

75  write(outputType: PptxGenJS.JSZIP_OUTPUT_TYPE): Promise<string | ArrayBuffer | Blob | Buffer | Uint8Array>
```

**Comment by adt-saksham** [comment]
Please refer this:
https://stackoverflow.com/questions/38875401/getting-error-ts2304-cannot-find-name-buffer

It worked for me.

**Comment by gitbrent** [comment]
Already fixed via Issue #654

Will be available in the upcoming v3.3.0 release.

**Comment by aviralgoyal** [comment]
Hi! I need to start using the library is there any definitive timeline as to when v3.3.0 will be released?
